### PR TITLE
Model instrumentation

### PR DESCRIPTION
**Pull request by TapasSenapati**
## Describe your changes
Make changes to adapter to fetch model type and incremental strategy

## Internal Jira ticket number or external issue link
[Internal Jira](https://jira.cloudera.com/browse/DBT-426)

## Testing procedure/screenshots(if appropriate):
Tested by performing normal and incremental run on dbt model
image

## Checklist before requesting a review
- [X] I have performed a self-review of my code
- [X] I have formatted my added/modified code to follow pep-8 standards
- [X] I have checked suggestions from python linter to make sure code is of good quality.

### CODE REVIEW

**Review by tovganesh — DISMISSED** [review]
looks good, only update the event type

File: dbt/adapters/spark_livy/relation.py [hunk 1]
```diff
@@ -40,3 +50,14 @@ def render(self):
                 "include, but only one can be set"
             )
         return super().render()
+
+    def log_relation(self, incremental_strategy):
+        if self.type:
+            tracker.track_usage(
+                {
+                    "event_type": "dbt_impala_new_incremental",
```
Comment (tovganesh) [inline dbt/adapters/spark_livy/relation.py]: Please change event type

File: dbt/adapters/spark_livy/relation.py [hunk 2]
```diff
@@ -32,6 +33,15 @@ class SparkRelation(BaseRelation):
     def __post_init__(self):
         if self.database != self.schema and self.database:
             raise RuntimeException("Cannot set database in spark!")
+        if self.type:
+            tracker.track_usage(
+                {
+                    "event_type": "dbt_impala_model_access",
```
Comment (tovganesh) [inline dbt/adapters/spark_livy/relation.py]: Please change event type

**Comment by TapasSenapati** [comment]
> looks good, only update the event type

Good find. Fixed the typo.

**Review by tovganesh — APPROVED** [review]
looks fine after the change for event type

**Pull request merged**